.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseMergedExtractorConfiguration.cs

```diff
@@ -109,6 +110,12 @@ private static void CheckBlackList(ref List<ExtractResult> extractResults, strin
                     continue;
                 }
 
+                // for cases like "早" in "早稻田" and "晚" in "晚安"
+                if (SingleWordMOAndEVRegex.IsMatch(extractResult.Text))
```

Comment:
I don't think this design is make sense as we use specify regex to filter that single char entity. I think we can define a HashSet or Dictionary to filter the single char entities.